### PR TITLE
ansible: document .ssh manual setup for release

### PR DESCRIPTION
**Pull request by sam-github**

### CODE REVIEW

**Comment by sam-github** [comment]
Discovered while investigating ci-release failure: https://ci-release.nodejs.org/job/iojs+release/4970/nodes=rhel7-s390x-release/console

cf. https://github.com/nodejs/build/issues/2080

```
08:47:41 ssh node-www "mkdir -p nodejs/custom/v12.14.1-rc.0"
08:47:41 ssh: Could not resolve hostname node-www: Name or service not known
08:47:41 Makefile:1139: recipe for target 'binary-upload' failed
08:47:41 make: *** [binary-upload] Error 255
08:47:41 Build step 'Conditional steps (multiple)' marked build as failure
```

**Comment by sam-github** [comment]
ci: https://ci-release.nodejs.org/job/iojs+release/4971/

File: ansible/MANUAL_STEPS.md [hunk 1]
```diff
@@ -1,4 +1,17 @@
-# Manual steps required to run ansible on machines
+# Manual steps required to setup machines
+
+## `release-*` machines
+
+Once setup, they must have `~iojs/.ssh` cloned from another machine, so they
+have the ssh setup and keys required to upload release artifacts to the
+nodejs.org web server.
```
Comment (rvagg) [inline ansible/MANUAL_STEPS.md]: Probably should list exactly what is needed: (1) a `config` containing a `Host node-www` and (2) a `node-www` private key.

**Comment by rvagg** [comment]
thanks for doing this, long overdue!

**Comment by sam-github** [comment]
take a look at the fixup. @rvagg

**Pull request merged**